drivers/sensor/bosch/bme680/bme680.c

```diff
@@ -444,12 +438,43 @@ static int bme680_init(const struct device *dev)
 		return err;
 	}
 
-	err = bme680_reg_write(dev, BME680_REG_CTRL_MEAS,
-			       BME680_CTRL_MEAS_VAL);
+	return bme680_reg_write(dev, BME680_REG_CTRL_MEAS, BME680_CTRL_MEAS_VAL);
 
 	return err;
 }
 
+static int bme680_pm_control(const struct device *dev, enum pm_device_action action)
+{
+	int rc = 0;
+
+	switch (action) {
+	case PM_DEVICE_ACTION_SUSPEND:
+	case PM_DEVICE_ACTION_RESUME:
+	case PM_DEVICE_ACTION_TURN_OFF:
```

Comment:
Should we not set the device to sleep mode in suspend? Otherwise I'm failing to see the value in adding PM to this device.

Reply:
What sleep mode do you suggest? The sensor automatically returns to sleep mode after a measurement is triggered
![image](https://github.com/user-attachments/assets/a71e18c5-d830-4740-bb23-af7bc0eb9695)
The point of this PR is the power domain support.

Reply:
Hmmm - I got thrown off by this table, which suggests sleep current and stand-by current are two separate things.
![image](https://github.com/user-attachments/assets/f49cc856-a60e-42ad-8240-2be9f9d5af9c)

This, however, may be a copy-over from the BME280, which does indeed have a Normal mode (periodic readings). That's where my mind was.